source/manage/cloud-vpc-private-connectivity.rst

```diff
@@ -0,0 +1,33 @@
+Cloud VPC Private Connectivity
+===============================
+
+.. include:: ../_static/badges/ent-cloud-only.rst
+   :start-after: :nosearch:
+
+VPC Private Connectivity (Private Link) offers Enterprise Cloud customers tailored solutions for private connectivity needs with Mattermost. These options enable customers to access Mattermost through AWS's network without using the public internet, or allow the Mattermost Infrastructure team to manage a Mattermost workspace hosted in the customer's VPC via an EKS cluster.
```

Comment:
It also provides the ability for customers to connect from Mattermost Cloud to their private webhook/endpoint/integration right?